clientd3d/d3drender.h

```diff
@@ -178,4 +178,9 @@ bool ShouldRenderInCurrentPass(bool transparent_pass, bool isTransparent);
 float FovHorizontal(long width);
 float FovVertical(long height);
 
+bool isFogEnabled();
+const long getShadeAmount();
+const Vector3D getSunVector();

```

Comment:
Should probably return as a const reference; this will copy the vector every time it is called.